UI/src/app/_components/settings/settings.component.html

```diff
@@ -47,7 +47,7 @@ <h2>
                     </table>
                 </div>
                 <div class="alert alert-info mt-3" role="alert" *ngIf="getNetworkDiscoveryList().length > 0">
-                    Found {{ getNetworkDiscoveryList().length }} new sources in your network.
+                    Found {{ getNetworkDiscoveryList().length }} new {{ getNetworkDiscoveryList().length === 1 ? 'source' : 'sources' }} in your network.
```

Comment:
I recently read that it's bad practice to call functions in an angular template... however, as it was already there, we could probably leave it that way. I just thought I'd mention it since we're now calling that function at least twice as often.